### PR TITLE
Adds lookback configuration for 5, 15m and defaults to 15

### PR DESCRIPTION
**Pull request by codefromthecrypt**
See https://github.com/openzipkin/zipkin/issues/2772

### CODE REVIEW

File: zipkin-server/pom.xml [hunk 1]
```diff
@@ -98,10 +105,11 @@
 
     <!-- Static content for the Classic UI -->
     <dependency>
-      <groupId>io.zipkin.java</groupId>
-      <artifactId>zipkin-ui</artifactId>
-      <version>2.12.10</version>
+      <groupId>com.github.openzipkin</groupId>
+      <artifactId>zipkin-classic</artifactId>
+      <version>15-minutes-of-fame-SNAPSHOT</version>
```
Comment (zeagord) [inline zipkin-server/pom.xml]: Can't beat you in naming versions 🗡

**Comment by codefromthecrypt** [comment]
Thanks! I verified everything including defaults and also visually verified the constructed urls

**Comment by codefromthecrypt** [comment]
still works

**Comment by codefromthecrypt** [comment]
Thanks tons, @zeagord!

**Pull request merged**